app/views/external_users/advocates/claims/edit.html.haml

```diff
@@ -2,10 +2,4 @@
 
 = render partial: 'layouts/header', locals: { page_heading: t('page_headings.edit_agfs_claim') }
 
-- present(@claim) do |claim|
-  .grid-row.grid-sidebar
-    .column-two-thirds.first
-      = render partial: 'form', locals: { claim: claim }
-
-    .column-one-third.first
-      = render partial: 'external_users/claims/summary_claims_sidebar', locals: { claim: claim, display_buttons: false }
+= render partial: 'external_users/claims/layout_proxy', locals: { claim: @claim, display_buttons: false }
```

Comment:
**[not a blocker/nit pick]** I find this name a bit confusing since it doesn't tell me its even related to the form rendering. 

💭  Would `external_users/claims/form_wrapper` or `external_users/claims/form_layout` be more clear of the intent of that _template_?

Reply:
I knew you were going to pick that up - did not feel like the right name. 

I have changed it to `form_layout_wrapper` - that should cover the intent of the template